src/asciidocParser.ts

```diff
@@ -110,7 +110,7 @@ export class AsciidocParser {
 
   // Load
 
-  public static load (textDocument: SkinnyTextDocument): { document: Asciidoctor.Document, baseDocumentIncludeItems: IncludeItems } {
+  public static loadWithoutParsingIncludes (textDocument: SkinnyTextDocument): { document: Asciidoctor.Document, baseDocumentIncludeItems: IncludeItems } {
```

Comment:
Since we don't use `document`, I think we could rename this function to `getBaseDocumentIncludeItems`.

Reply:
Never mind, we are actually using `document.getSourceLines()` 😬 

Having said that, I still think that we should rename this function.
Also, I wonder if `document.getSourceLines()` is actually the same as `textDocument.getText().split('\n')`. If that's the case then we don't need to return `document`.

Reply:
used `textDocument.getText().split('\n')` and tests are passing